Frontend/voitheia-App/src/app/components/about/about-us/about-us.component.html

```diff
@@ -1 +1,70 @@
-<p>about-us works!</p>
+
+<!-- Team Container -->
+<div class="container-fluid text-center text-justify">
+  <h1> {{'AboutUs.Texts.AboutUs' | translate }} </h1>
+  <p> {{'AboutUs.Texts.Text1' | translate }} </p>
+
+  <!--Row 1: Names-->
+  <div class="pt-4 row justify-content-center">
+    <br>
+    <div class="col-md-3 text-center w-100">
+      <img src="/assets/images/Team_Lukas.jpg" alt="Lukas" class="rounded-circle mx-auto d-block" width="120" height="120">
+      <h4>Lukas</h4>
+      <p>{{'AboutUs.Texts.Lukas' | translate }}</p>
+      <h6 class="tasks"> Backend-Entwicklung <br /> Data Security</h6>
+    </div>
+
+    <div class="col-md-3 text-center">
+      <img src="/assets/images/Team_Andi.jpg" alt="Andi" class="rounded-circle mx-auto d-block" width="120" height="120">
+      <h4>Andi</h4>
+        <p>{{'AboutUs.Texts.Andi' | translate }}</p>
+        <h6> Backend-Entwicklung</h6>
+      </div>
+
+    <div class="col-md-3 text-center">
+      <img src="/assets/images/Team_Sebastian.jpg" alt="Sebastian" class="rounded-circle mx-auto d-block" width="120" height="120">
+      <h4>Sebastian</h4>
+        <p>{{'AboutUs.Texts.Sebastian' | translate }}</p>
+        <h6> Cloud-Connection </h6>
+      </div>
+
+    <div class="col-md-3 text-center">
+      <img src="/assets/images/Team_Benedikt.jpg" alt="Benedikt" class="rounded-circle mx-auto d-block" width="120" height="120">
+      <h4>Benedikt</h4>
+        <p>{{'AboutUs.Texts.Benedikt' | translate }}</p>
+        <h6> Frontend-Entwicklung</h6>
+      </div>
+    </div>
+
+  <!--Row 2 -->
+  <div class="pt-4 row justify-content-center">
+    <br>
+    <div class="col-md-3 text-center">
+      <img src="/assets/images/Team_Kevin.jpg" alt="Kevin" class="rounded-circle mx-auto d-block" width="120" height="120">
+      <h4>Kevin</h4>
+      <p>{{'AboutUs.Texts.Kevin' | translate }}<br /></p>
+      <h6> Backend-Entwicklung <br />Frontend-Entwicklung <br /> Design </h6>
+    </div>
+
+    <div class="col-md-3 text-center">
+      <img src="/assets/images/Team_Christoph.jpg" alt="Christoph" class="rounded-circle mx-auto d-block" width="120" height="120">
+      <h4>Christoph</h4>
+      <p>{{'AboutUs.Texts.Christoph' | translate }}<br />  </p>
+      <h6> App Development</h6>
+    </div>
+
+    <div class="col-md-3 text-center">
+      <img src="/assets/images/Team_Helen.jpg" alt="Helen" class="rounded-circle mx-auto d-block" width="120" height="120">
```

Comment:
The placeholder image is still copied several times. Should be present once and used several times.